src/components/Input/styled/helpText.js

```diff
@@ -1,17 +1,17 @@
 import styled from 'styled-components';
 import { MARGIN_X_SMALL } from '../../../styles/margins';
-import { FONT_SIZE_TEXT_MEDIUM } from '../../../styles/fontSizes';
+import { FONT_SIZE_TEXT_SMALL } from '../../../styles/fontSizes';
```

Comment:
you cannot change the style of another component without discussing it with Tahimi

Reply:
@yvmunayev I approve this change, this is part of the task :)